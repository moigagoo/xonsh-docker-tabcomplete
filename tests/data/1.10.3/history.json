{
        "arguments": [
                "-H", "--human=true",
                "--help",
                "--no-trunc",
                "-q", "--quiet"
        ],
        "commands": []
}
