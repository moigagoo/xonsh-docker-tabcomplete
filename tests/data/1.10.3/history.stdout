
Usage:	docker history [OPTIONS] IMAGE

Show the history of an image

  -H, --human=true    Print sizes and dates in human readable format
  --help              Print usage
  --no-trunc          Don't truncate output
  -q, --quiet         Only show numeric IDs
